.gitignore

```diff
@@ -1,3 +1,4 @@
 env
 config.json
 pid
+tg-bot/*
```

Comment:
What's the use of this directory?

Reply:
This was a venv. I have deleted it and removed the ```dir``` from .gitignore.
I have squashed the commits.
Please review this PR.